moped-editor/src/queries/signals.js

```diff
@@ -17,8 +17,10 @@ export const SIGNAL_PROJECTS_QUERY = gql`
             }
             status_id: { _neq: 4 }
           }
-        ]
+        ],
+        status_id: {_eq: 1}
```

Comment:
This line seems to be the rabbit hole that Amenity's question would take you down.